data-raw/dm_ped.R

```diff
@@ -0,0 +1,74 @@
+# Dataset: dm_ped
+# Description: Create DM test SDTM dataset for pediatric studies
+
+# Create DM for pediatric  ----
+
+## Read in original data ----
+data("dm")
```

Comment:
Like @manciniedoardo mentioned, it'll be great to load the required packages. Here we need `pharmaversesdtm` in order to use `dm`

Reply:
Sorry, I didn't look at it from that bigger scope, but sure it makes sense. As per @manciniedoardo recommendation, if you plan to add temporary and then remove, it'll be easily to find such temporary lines if you put a `# TODO remove...` (Rstudio will recognize the TODO as a tag)

Reply:
thanks - i have added ```library(pharmaversesdtm) #TODO remove when this script is moved to pharmaversesdtm```

Reply:
as a follow-up: adding this line of code results in a lint error in the CI/CD workflow: 
```
$message
[1] "Function \"library\" is undesirable. As an alternative, use roxygen2's @importFrom statement in packages and `::` in scripts, instead of modifying the global search path."
$line
[1] "library(pharmaversesdtm) # TODO remove when this script is moved to pharmaversesdtm"
```
How can i solve this issue? @manciniedoardo would you know?

Reply:
Is this only happening locally? I can't see it as an error here in the PR?

Reply:
This is in the PR: https://github.com/pharmaverse/admiralpeds/actions/runs/8282755813/job/22664330922?pr=23
and locally  (actually this applies to all libraries)
![image](https://github.com/pharmaverse/admiralpeds/assets/57490460/f5308528-d655-4ee3-a8ba-d4f542e54fc1)

Reply:
I think we can make an exclusion in the `.lintr` file, just add the `data-raw` path to here, the lintr to check for "library" function call is mostly for codebase/testthat files
![image](https://github.com/pharmaverse/admiralpeds/assets/43018762/40dca499-b482-4891-b5a3-fbc58364499b)

Reply:
thank you - i have updated the ```.lintr``` file and the error has disappeared 👍